x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/transport/nio/SSLDriverTests.java

```diff
@@ -178,7 +179,11 @@ public void testHandshakeFailureBecauseProtocolMismatch() throws Exception {
         } else {
             serverProtocols = new String[]{"TLSv1.2"};
             clientProtocols = new String[]{"TLSv1.1"};
-            expectedMessage = "The client supported protocol versions [TLSv1.1] are not accepted by server preferences [TLS12]";
+            if (inZuluJvm) {
+                expectedMessage = "No appropriate protocol (protocol is disabled or cipher suites are inappropriate)";
+            } else {
+                expectedMessage = "The client supported protocol versions [TLSv1.1] are not accepted by server preferences [TLS12]";
+            }
```

Comment:
I don't think we should be worrying about checking the vendor for these sorts of fixes.

Why do we care about which message appears on which JVMs? Aren't both acceptable on all JDKs?

I'd prefer that, where possible, these sort of fixes end up with assertions that effectively say "any of these message are acceptable, the exact message might vary by JVM vendor/version"

Reply:
Indeed, I noticed this same test in 7.x branch uses `anyOf` when backporting. I thought about whether I should add this new error message to it. But didn't do so because it's already done this way in the `master` branch. I'll keep this in mind and change it to use `anyOf` if it fails again in future.